en/docs/learn/exporting-user-data-from-wso2-is.md

```diff
@@ -0,0 +1,28 @@
+#Exporting User Data from WSO2 Identity Server
+
+1. Download **scim-bulk-user-export-tool** from [here]().
```

Comment:
update the link in ()

Reply:
Fixed in https://github.com/wso2/docs-is/pull/1761/commits/ad0efb308d19d057c99ff2563d643ef85f19bb6f